controllers/dbaas.redhat.com/configmap.go

```diff
@@ -0,0 +1,101 @@
NjIuNjctMTYuNDJjNi4zNCwwLjM2LDEyLjY2LDEuNDksMTguOTEsMi43OWMtMi4wNSwwLjQ5LTExLjQ1LDIuOTYtMTQuMzYsNy41NwoJCQkJCWMtMC45OSwxLjc2LTIuNjUsMy42My0zLjAzLDUuNzRjLTAuMjcsMS40OCwxLjIyLDMuMjgsMS45Miw0Ljk0YzEuMjQtMC45NiwyLjUtMS44OSwzLjctMi44OWMwLjI4LTAuMjMsMC4zOC0wLjcxLDAuNS0xLjA5CgkJCQkJYzEuNzItNS4zNiw2LjE2LTcuOTUsMTAuODYtMTAuOThjMC40MywyLjEsMC42MSwzLjc5LDEuMTMsNS4zOGMwLjcsMi4xNi0wLjI2LDMuNTgtMS44OCw0LjYyYy0wLjgyLDAuNTMtMS45NiwwLjU2LTIuOTYsMC44MgoJCQkJCWMtMC4yLTEuMTEtMC41Ni0yLjIyLTAuNTMtMy4zMmMwLjAyLTAuOSwwLjUxLTEuNzksMC43OS0yLjY4Yy00Ljk4LDMuMjMtNi44OCw4LjItOC43MywxMy40MWMxLjE4LDAuNDIsMi4wMywwLjcyLDIuMzIsMC44MgoJCQkJCWMtMC41OCwxLjg5LTEuMTMsMy42OC0xLjcsNS41NGw0LjMxLTIuMTNjMi44Ni0xLDUuMDQtMS43Myw3LjItMi41M2MzLjM0LTEuMjQsNC45NC0zLjc3LDQuOTYtNy4yNAoJCQkJCWMwLjA0LTUuNDktMS42LTEwLjU2LTMuOTktMTUuODdjMC43OCwwLjE2LDEuNTYsMC4yOCwyLjM0LDAuNDVjNy42OSwxLjY4LDE0LjEyLDYuMDQsMTkuNTQsMTEuNjcKCQkJCQljNC4yNyw0LjQ0LDguNzgsOC44MywxMi4xNywxMy45MWM3Ljc2LDExLjYxLDE4LjQ0LDIwLjc3LDI2LjA5LDMyLjQ4YzEzLjgyLDIxLjE0LDI2LjEzLDQzLjAzLDM1LjIsNjYuNjUKCQkJCQljMC4zOSwxLjAxLDAuNzIsMi4wNSwxLjEyLDMuMThjLTIuOTQsMC45My01LjcyLDEuOC04LjQzLDIuNjVjMTcuNzgsNy4zMSwzMC4wOCwyMS4zOSw0Mi45OSwzNC41NQoJCQkJCWMzLjM2LDMuNDMsNi41Myw3LjAyLDkuNjEsMTAuNjdjMC44NC0zLjAyLDEuNjYtNi4wNiwyLjM1LTkuMTRDMzQ4Ljg3LDIwOS4xMiwzMzcuMjUsMTk3Ljk5LDMyNC44MywxODcuNTh6IE0xNjMuMTcsMjYyLjM0CgkJCQkJYzIwLjY4LDIuMzQsNDAuOTYsMS40Miw1OC42Mi0xMS41MWM3LjI0LTUuMywxMy45LTExLjM4LDIwLjg3LTE3LjA2YzEuMDYtMC44NiwyLjI3LTEuNzUsMy41NS0yLjA3CgkJCQkJYzguODYtMi4xOSwxNi4yNi02LjU0LDIxLjYyLTE0LjA4YzAuMTUtMC4yMSwwLjQzLTAuMzIsMS4yOS0wLjk0Yy0wLjg5LDIuNjItMS4xOSw0Ljg0LTIuMzEsNi41Yy0xLjgxLDIuNjgtMi4yMSw1LjUyLTIuNCw4LjU4CgkJCQkJYy0xLjA1LDE2LjczLTIuOTIsMzMuMjctMTEuNTQsNDguMThjLTAuMSwwLjE3LTAuMiwwLjM1LTAuMjgsMC41NGMtNS42LDEzLjU2LTE2LjUsMTguMzEtMzAuMywxOC4xNAoJCQkJCWMtMTAuMjktMC4xMy0yMC4yNS0yLjQtMzAuMTMtNWwtNTUuMzEtMTQuNzNjLTYuNDQtMi42OS02Ljk5LTQuMDYtNC4xMi0xMC4yM2MyLjk4LTYuNCw2LjAyLTEyLjc3LDkuMjQtMTkuNTkKCQkJCQlDMTQ1Ljk2LDI1OC45NiwxNTQuNCwyNjEuMzUsMTYzLjE3LDI2Mi4zNHogTTg3LjM1LDI0Ny40NWM4LjU5LTEwLjQ0LDguNC0xMS4wNCwyMS44MS0xMS40MmM5LjUyLTAuMjcsMTkuMDksMC45MSwyOS4yOSwxLjQ5CgkJCQkJYy0wLjg1LDEuOTktMS4yOCwzLjA5LTEuNzgsNC4xNWMtNC44MywxMC4zNC05LjczLDIwLjY2LTE0LjQ3LDMxLjA0Yy0wLjg4LDEuOTMtMS44LDIuMzQtMy45LDEuOTkKCQkJCQljLTYuMzktMS4wNi0xMi42Ny0wLjYtMTkuMDQsMi44MWMxLjYzLDAuMiwyLjg3LDAuMTEsMy45MiwwLjUzYzIuNjQsMS4wNyw1LjY2LDEuODUsNy42NywzLjY5YzMuNTksMy4yOCw0LjQ5LDYuOTMsMi40MywxMi4zMQoJCQkJCWMtNS42OSwxNC44OC0xMC4yNiwzMC4xOS0xNS4yNyw0NS4zM2MtMC4xOCwwLjU2LTAuMzYsMS4xMi0wLjU0LDEuNjdjLTAuMzIsMC4xLTAuNjQsMC4yMS0wLjk2LDAuMzIKCQkJCQljLTMuMjctNy4xNC02LjcyLTE0LjE5LTkuNzUtMjEuNDNjLTUuMTMtMTIuMjUtOC45LTI0Ljg5LTEwLjIxLTM4LjJDNzUuMjksMjY4LjcxLDc5LjA5LDI1Ny41LDg3LjM1LDI0Ny40NXoiLz4KCQkJCTxwYXRoIGNsYXNzPSJzdDIiIGQ9Ik0xNTcuNTYsMTAxLjljMi45MS0yLjcxLDUuNDMtNC4yNyw3LjA0LTUuMWMtMC4zOSwyLjEzLDEuNyw1LjY5LDUuOTUsNi4yMmM3Ljk2LDAuMTEsNy44OC05LjIzLDcuODgtOS4yMwoJCQkJCWMyLjA2LTAuNTUsNC40Ni0wLjc5LDQuNDYtMC43OUMxNjYuMTksODUuMDIsMTU3LjU2LDEwMS45LDE1Ny41NiwxMDEuOXoiLz4KCQkJPC9nPgoJCTwvZz4KCTwvZz4KPC9nPgo8L3N2Zz4K
```

Comment:
Did we just borrow the Crunchy OLM image val here and need to run by the Crunchy eng team?

Reply:
right it's copied from operator CSV file.